.changeset/fix-reown-branding.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We'd need to create change log including all packages. When running `pnpm changeset`;

- Select all packages both changed and not changed
- Skip major
- Skip minor
- Give a description like our convention for changesets, like "Fixes wui-ux-by-reown to make it always displayed"